src/EthereumClassicNameService.js

```diff
@@ -8,11 +16,15 @@ class EthereumClassicNameService {
 
   async getResolver(name) {
     // TODO using web3 to fetch data
-    console.log(name);
+    registryInit(this.restURL);
+    const resolverAddr = await getResolver(name);
+    return resolverAddr;
   }
 
-  async getAddress(name) {
-    
+  async getAddress(name, resolverAddr) {
```

Comment:
This function can use more easily just fill in `name`, we can get the `resolver` from `getResolver` function and response the `address`